mods/cnc/rules/defaults.yaml

```diff
@@ -651,6 +659,8 @@
 ^Ship:
 	Inherits@1: ^ExistsInWorld
 	Inherits@3: ^SpriteActor
+	UpdatesPlayerStatistics:
+		AddToArmyValue: true
```

Comment:
The only ship in TD is the gunboat, which is an unconrtollable support unit like the a10.  Since the a10 doesn't count, this shouldn't either.

Reply:
Dropped.

Reply:
Dropped.